### DISCUSSION TITLE
URGENT:Github Account Suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#139631] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Its more than a month my old github account has been suspended. I was working in a hackathon project and suddenly i noticed my account has been suspended, I raised a ticket on github customer support but till now i didnt receive any mail from them.
As a college student & a programmer github is vital. It would be really helpful if someone can help me with this 

Ticket id : 2951873
Old account username : Rudra-Sankha-Sinhamahapatra
Date of ban : 20th August 2024

Thanks

**Comment by ghost** [comment]
creating a temporary account can serve as a short-term solution until the suspended account is restored or If your suspended account was a collaborator in any repositories, and you had other collaborators or teammates, you can request them to share or transfer ownership of the repositories to your temporary account. This way, you can continue working on your projects without losing access to the important files.

**Comment by LiteBrite82** [comment]
Hi @rushhaaland 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
